cairo-sys-rs/src/lib.rs

```diff
@@ -13,10 +13,10 @@ extern crate glib_sys as glib_ffi;
 #[cfg(any(feature = "xlib", feature = "dox"))]
 extern crate x11;
 
-#[cfg(windows)]
+#[cfg(all(windows, feature = "win32-surface"))]
 extern crate winapi as winapi_orig;
 
-#[cfg(windows)]
+#[cfg(all(windows, feature = "win32-surface"))]
```

Comment:
IMHO also need to add "dox" here and to previous cfg

Reply:
No, I wrong, "dox" in Cargo.toml not needed "winapi" as it will fail on nonwindows.
And here need separate module for it
```
#[cfg(all(feature = "dox"))]
pub mod winapi {
    pub enum _HDC {}
        pub type HDC = *mut _HDC;
}
```

Reply:
Maybe with and `and(.., not(all(windows, feature = "win32-surface"))` but IMHO this unneeded as docs will be generated on unix anyway.